### DISCUSSION TITLE
How to get instance`s location?

### DISCUSSION THREAD

**Discussion by NZheludkov** [The-OpenROAD-Project/OpenROAD#4981] [General]
I would like to get location for some instances in the design by command in terminal to use it in my flow script. Is any possible options to get x and y location of certain instance by command?

**Comment by maliberty** [comment]
There is an example at https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/2c7954aab840678143914048d53dec3846026794/flow/scripts/add_routing_blk.tcl#L22

**Reply by NZheludkov** [reply]
Thanks a lot!
